src/libstd/path.rs

```diff
@@ -1324,11 +1368,14 @@ impl cmp::Ord for Path {
 }
 
 /// Freely convertible to a `Path`.
+#[stable(feature = "rust1", since = "1.0.0")]
 pub trait AsPath {
```

Comment:
My impression was that this was one of the motivating reasons for generic conversion traits, so yeah we probably want to wait for that.